iOS/SampleApp/SampleApp/Services/ManagingTextViews.swift

```diff
@@ -0,0 +1,69 @@
+// Copyright 2019 Flying Vehicle Monster team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import UIKit
+
+extension ViewController : UITextViewDelegate {
```

Comment:
And another one, I'm just gonna copy my previous comment 
 we don't need to handle any alerts when returning to sample app, it doesn't need to support anything else than opening FVM and printing FVM outcome to console